spec/factories/user.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :user do
-    email "test@test.ru"
-    password "test123"
+    email 								"test@test.ru"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.